src/modules/mavlink/mavlink_parameters.h

```diff
@@ -161,6 +163,6 @@ class MavlinkParametersManager
 
 	Mavlink &_mavlink;
 
-	bool _first_send{false};
+	bool _first_send{true};
```

Comment:
This still isn't very clear. I'd formulate it so it's clearer, what about:


```suggestion
	bool _should_send_first{true};
```

Reply:
Ok, then I'd call it:

```suggestion
	bool _is_sending_param_first{true};
```